features/dd-sdk-android-trace/src/main/java/com/datadog/trace/api/GlobalTracer.java

```diff
@@ -5,12 +5,19 @@
 import java.util.ArrayList;
 import java.util.Collection;
 
+import kotlin.Deprecated;
+import kotlin.DeprecationLevel;
+
 /**
  * A global reference to the registered Datadog tracer.
  *
  * <p>OpenTracing's GlobalTracer cannot be cast to its DDTracer implementation, so this class exists
  * to provide a global window to datadog-specific features.
  */
+@Deprecated(
+        message = "Use [io.opentracing.util.GlobalTracer] instead, this class will be removed soon",
+        level = DeprecationLevel.ERROR
+)
```

Comment:
you can also utilize `replaceWith` argument.

Reply:
updated